docs/spec/spec.md

```diff
@@ -25,21 +25,21 @@ broker._
 
 #### Spec
 
-| Field        | Type                    | Description                                                                                                                                                                | Constraints |
-| ------------ | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
-| broker       | String                  | Broker is the broker that this trigger receives events from. Defaults to 'default'.                                                                                        |             |
-| filter       | TriggerFilter           | Filter is the filter to apply against all events from the Broker. Only events that pass this filter will be sent to the Subscriber. Defaults to subscribing to all events. |             |
-| subscriber\* | eventing.SubscriberSpec | Subscriber is the addressable that receives events from the Broker that pass the Filter.                                                                                   |             |
+| Field        | Type          | Description                                                                                                                                                                | Constraints |
+| ------------ | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| broker       | String        | Broker is the broker that this trigger receives events from. Defaults to 'default'.                                                                                        |             |
+| filter       | TriggerFilter | Filter is the filter to apply against all events from the Broker. Only events that pass this filter will be sent to the Subscriber. Defaults to subscribing to all events. |             |
+| subscriber\* | Destination   | Subscriber is the object resolved into a uri that receives events from the Broker that pass the Filter.                                                                    |             |
```

Comment:
'object resolved into a uri' feels inaccurate. Can't it be either an object that is _resolvable_ into a URI or just a URI itself?

Reply:
it can be ref, ref+uri, or uri internally and is intended to be resolved to a single uri is what I am trying to say.